build-and-test.ps1

```diff
@@ -1,19 +1,57 @@
+[cmdletbinding()]
+param(
+    [Parameter(Mandatory=$true)]
+    [ValidateSet("win", "linux")]
+    [string]$Platform,
```

Comment:
I would prefer to not have a default because using the default in a linux environment wastes a lot of time because it triggers the nano images to be pulled which fails but not immediately.